packages/waku/src/router/fs-router.ts

```diff
@@ -25,9 +25,9 @@ export function fsRouter(
           { join, dirname, extname, sep },
           { fileURLToPath },
         ] = await Promise.all([
-          import(DO_NOT_BUNDLE + 'node:fs/promises'),
-          import(DO_NOT_BUNDLE + 'node:path'),
-          import(DO_NOT_BUNDLE + 'node:url'),
+          import(/* @vite-ignore */ DO_NOT_BUNDLE + 'node:fs/promises'),
+          import(/* @vite-ignore */ DO_NOT_BUNDLE + 'node:path'),
+          import(/* @vite-ignore */ DO_NOT_BUNDLE + 'node:url'),
```

Comment:
I somehow thought it didn't cause issues. https://github.com/dai-shi/waku/pull/792/files#diff-dbb54d801e436acf21fb8afff808d82c614f9f0a7fc5e3ce8a666e437eef4d04
I think I missed something.

Reply:
Yea it does, so many warnings!